tests/protobuf-test.py

```diff
@@ -34,7 +38,9 @@ def test_clear_field(self):
     def test_unknown_field_ignored(self):
         # Protocol Buffers ignore unknown fields during deserialization.
         serialized = self.person.SerializeToString()
-        corrupted = serialized + b"\x20\x01"  # Append unknown field (field number 4, varint 1)
+        corrupted = (
+            serialized + b"\x20\x01"
+        )  # Append unknown field (field number 4, varint 1)
```

Comment:
Is there any particular reason for this change? It's more readable this way; I was just wondering if it was done by an automated formatter or manually.